src/clarity-angular/tabs/_vertical-tabs.clarity.scss

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2016 - 2017 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import "../utils/variables.clarity";
```

Comment:
We don't need these imports here now. Everything has been moved into `main.scss`